zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/TestHelper.java

```diff
@@ -32,7 +32,7 @@
 
 public class TestHelper {
   private static final Logger LOG = LoggerFactory.getLogger(TestHelper.class);
-  public static final long WAIT_DURATION = 20 * 1000L; // 20 seconds
+  public static final long WAIT_DURATION = 60 * 1000L; // 20 seconds
```

Comment:
1. the comment is not updated accordingly.
2. Could you please justify the reason we need to triple the wait duration?

Reply:
Github zookeeper is slow. See #1268 for profiling result. Using 60 secs does not really hurt anything, as we still poll the condition every 500ms. 

Otherwise, the test can be flaky.

Reply:
"the comment is not updated accordingly."

Reply:
Good point, will update this one. Let us first resolve the bigger issue about how to initialize and how to make test stable.